src/chemistry/utils/aircraft_emit.F90

```diff
@@ -208,7 +208,10 @@ subroutine aircraft_emit_init()
           !-----------------------------------------------------------------------
           forcings_air(m)%file%stepTime    = .true.  ! Aircraft data is not to be interpolated in time
           forcings_air(m)%file%cyclical_list    = .true.  ! Aircraft data cycles over the filename list
-          forcings_air(m)%file%weight_by_lat     = .true.  ! Aircraft data -  interpolated with latitude weighting
+!+++ARH
+          !forcings_air(m)%file%weight_by_lat     = .true.  ! Aircraft data -  interpolated with latitude weighting
```

Comment:
Why was this default set to .false.?  Also could you remove all the !+++ARH comment strings.

Reply:
I believe ```.true.``` gives a failure when turning on aircraft emissions w/ dycores on unstructured grids. I can remove it here and make sure not run with this feature. Cecile's not even running w/ this feature.

@chihchen24 @andrewgettelman are you aware of this issue? That ```forcings_air(m)%file%weight_by_lat     = .true.``` fails for unstructured grids?

Reply:
Got it. Thanks.